config/database.yml

```diff
@@ -9,27 +9,19 @@
 # And be sure to use new-style password hashing:
 #   http://dev.mysql.com/doc/refman/5.0/en/old-client.html
 development:
-  adapter: mysql2
-  encoding: utf8mb4
-  database: abtt_development_master
+  adapter: sqlite3
+  database: db/development.sqlite3
   pool: 5
-  username: deploy-abtech-tracker
-  password:
-  socket: /var/run/mysqld/mysqld.sock
-  collation: utf8mb4_bin
+  timeout: 5000
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  adapter: mysql2
-  encoding: utf8
-  database: abtt_test
+  adapter: sqlite3
+  database: db/test.sqlite3
   pool: 5
-  username: deploy-abtech-tracker
-  password:
-  socket: /var/run/mysqld/mysqld.sock
-  collation: utf8_unicode_ci
+  timeout: 5000
```

Comment:
We haven't run tests in a while so this is really a nitpick, but ideally we have a way to run tests on how the server deploys as well (maybe a `dev-test` and `test`).